frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/grid/grid_menu_item.dart

```diff
@@ -6,14 +6,14 @@ import 'package:appflowy_editor/appflowy_editor.dart';
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 
-SelectionMenuItem gridMenuItem = SelectionMenuItem(
+SelectionMenuItem referenceGridMenuItem = SelectionMenuItem(
   name: LocaleKeys.document_plugins_referencedGrid.tr(),
   icon: (editorState, onSelected, style) => SelectableSvgWidget(
-    name: 'editor/board',
+    name: 'editor/grid',
     isSelected: onSelected,
     style: style,
   ),
-  keywords: ['referenced', 'grid'],
+  keywords: ['referenced', 'grid', 'database'],
```

Comment:
Should these keywords be internationalized?

Reply:
I think no. the keyword doesn't need to be internationalized now.